Consolidation OOP/consolidate/mesher/meshes.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pdb
+
+class Meshes():
```

Comment:
Please rename this class and file `TwoPlates`. We will later try to add a class in the `mesher` folder called `Plate` (which will be the mesh of a single plate). We can add other shapes later.